RSE Manager: make the number of stat retries configurable Fix #1833

berghaus opened the pull request:
RSE Manager: make the number of stat retries configurable Fix #1833

Commit:
RSE Manager: retry stat multiple times before failing

When requesting file stats of a storage element immediately on the
competion of the upload sometimes this fails. The storage element may
not yet have completed internal operations causing a race condition. To
avoid this retry again with an increasing pause inbetween. After a
minute assume that the storage element should have completed it's
internal operations.

Commit:
RSE Manager: make the number of stat retries configurable Fix #1833

Configure the number of times the RSE manager will attempt to stat a
file before giving up with protocol_stat_retries in the client section.

bari12 commented on lib/rucio/rse/rsemanager.py:
```diff
@@ -713,6 +716,31 @@ def find_matching_scheme(rse_settings_dest, rse_settings_src, operation_src, ope
+    """
+    from time import sleep
+    try:
+        retries = config_get('client', 'protocol_stat_retries')
```
The `config_get` command already has a default option. Would shorten the code a bit to do that. Also the import should go to the top of the file. (I think this was already fixed in your previous PR, maybe in the re-creation as patch this got lost?)

Commit:
RSE Manager: add requested improvements Fix #1833

Move import to the top of the module. Use the default option of the
config_get function to streamline the configuration import.
